packages/mint-components/src/components/sqm-referral-codes/ReferralCodes.feature

```diff
@@ -0,0 +1,64 @@
+@author: sam
+@owner: sam
+
+Feature: Referral Codes Component
+
+    A paginated display of available and used promo codes for the user to share with others
+
+    @motivating
+    Scenario: Promo code can be shared using various share mediums
+        Given an authenticated user
+        And the program has promo codes configured
+        When the promo code section is loaded
+        Then the promo code can be copied directly
+        And the promo code can be shared using <shareMedium>
+
+        Examples:
+            | shareMedium |
+            | DIRECT      |
+            | EMAIL       |
```

Comment:
Just flagging that clicking on any of these share medium buttons needs to count as being copied